packages/esm-service-queues-app/src/routes.json

```diff
@@ -108,5 +103,27 @@
       "component": "activeVisitsRowActions",
       "slot": "queue-table-extension-column-slot"
     }
+  ],
+  "workspaces": [
+    {
+      "name": "service-queues-service-form",
+      "title": "addNewQueueService",
+      "component": "addnewQueueServiceWorkspace"
+    },
+    {
+      "name": "service-queues-room-form",
+      "title": "addNewQueueServiceRoom",
+      "component": "addnewQueueServiceRoomWorkspace"
```

Comment:
```suggestion
      "component": "addNewQueueServiceWorkspace"
    },
    {
      "name": "service-queues-room-form",
      "title": "addNewQueueServiceRoom",
      "component": "addNewQueueServiceRoomWorkspace"
```

nit: Pedantic.

Reply:
Not at all; software engineering is for pedants. Thanks for your careful reading.